mogoria/TravelMantics
# #167775147 Enable users save and retrieve images

## What does this PR do?
Enables users to view and retrieve images.

#### Description of Task to be completed?
- Enable users to retrieve image deals
- Enable admins to save image deals

#### How should this be manually tested?
- Clone this repo
- Check out this branch
- Run using an emulator or real device

 #### Any background context you want to provide?

[None]

#### What are the relevant pivotal tracker stories?
[#167775147](https://www.pivotaltracker.com/n/projects/2380934/stories/167775147)
